<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>boundaryPropertyDialog</class>
 <widget class="QDialog" name="boundaryPropertyDialog" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Body</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Properties</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="boundaryConditionLabel" >
        <property name="text" >
         <string>Boundary condition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boundaryConditionCombo" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="boundaryAsABody" >
     <property name="text" >
      <string>    Use as a body</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
       <property name="icon" >
        <iconset theme="list-add" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="discardButton" >
       <property name="text" >
        <string>Discard</string>
       </property>
       <property name="icon" >
        <iconset theme="list-remove" />
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
